Example/TokenTextViewExample/TokenTextViewExample/ViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ViewController.swift
+//  TokenTextViewExample
+//
+//  Created by Chris Lee on 8/11/22.
+//
+
+import UIKit
+
+class TokenTextViewExampleController: UIViewController {
+    @IBOutlet private var tokenTextView: TokenTextView! {
+        didSet {
+            tokenTextView.layer.borderColor = UIColor.black.cgColor
+            tokenTextView.layer.borderWidth = 1.0
+        }
+    }
+    @IBOutlet private var tokenListTableView: UITableView!
+
+    private let tokenList = MockDataLoader.loadMockTokenList() ?? []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        tokenTextView.tokenList = tokenList
+    }
+}
+
+extension TokenTextViewExampleController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        tokenList.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let tableViewCell = UITableViewCell()
+        tableViewCell.textLabel?.text = tokenList[indexPath.row].name
+        return tableViewCell
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let token = tokenList[indexPath.row]
```

Comment:
I think we should deselect the row too, otherwise it will stay highlighted 
![Simulator Screen Shot - iPhone 13 Pro - 2022-08-18 at 11 56 31](https://user-images.githubusercontent.com/23501183/185472777-826cec39-10cc-4366-884a-eca7a0bc6c88.png)

Reply:
[d05bb4e](https://github.com/open-turo/token-text-view/pull/15/commits/d05bb4e67831b1871781e31fc07e73935efcca74)